fuse_core/test/test_throttled_callback.cpp

```diff
@@ -151,33 +151,34 @@ class PointSensorModel
   /**
    * @brief Keep callback, that counts the number of times it has been called and caches the last message received
    *
-   * @param[in] msg A geometry_msgs::Point message
+   * @param[in] msg A geometry_msgs::msg::Point message
    */
-  void keepCallback(const geometry_msgs::Point::ConstPtr& msg)
+  // TODO(CH3): I'm iffy on this...
+  void keepCallback(const geometry_msgs::msg::Point::ConstSharedPtr msg)
```

Comment:
The callback signature for the ThrottledMessageCallback is `void(const typename M&)`. So shouldn't this be `void keepCallback(const geometry_msgs::msg::Point& msg)`?